packages/semantic-tokens/src/optional/tokens.ts

```diff
@@ -0,0 +1,622 @@
+  textRampLgSubsectionheaderFontsizeRaw,
+  textRampLgSubsectionheaderLineheightRaw,
+  textRampMetadataFontsizeRaw,
+  textRampMetadataLineheightRaw,
+  textRampPageheaderFontsizeRaw,
+  textRampPageheaderLineheightRaw,
+  textRampReadingbodyFontsizeRaw,
+  textRampReadingbodyLineheightRaw,
+  textRampSectionheaderFontsizeRaw,
+  textRampSectionheaderLineheightRaw,
+  textRampSmItembodyFontsizeRaw,
+  textRampSmItembodyLineheightRaw,
+  textRampSmItemheaderFontsizeRaw,
+  textRampSmItemheaderLineheightRaw,
+  textRampSmLegalFontsizeRaw,
+  textRampSmLegalLineheightRaw,
+  textRampSmMetadataFontsizeRaw,
+  textRampSmMetadataLineheightRaw,
+  textRampSmPageheaderFontsizeRaw,
+  textRampSmPageheaderLineheightRaw,
+  textRampSmReadingbodyFontsizeRaw,
+  textRampSmReadingbodyLineheightRaw,
+  textRampSmSectionheaderFontsizeRaw,
+  textRampSmSectionheaderLineheightRaw,
+  textRampSmSubsectionheaderFontsizeRaw,
+  textRampSmSubsectionheaderLineheightRaw,
+  textRampSubsectionheaderFontsizeRaw,
+  textRampSubsectionheaderLineheightRaw,
+  textStyleAiHeaderFontfamilyRaw,
+  textStyleAiHeaderLetterspacingRaw,
+  textStyleAiHeaderWeightRaw,
+  textStyleAiRegularFontfamilyRaw,
+  textStyleAiRegularLetterspacingRaw,
+  textStyleAiRegularWeightRaw,
+  textStyleArticleHeaderFontfamilyRaw,
+  textStyleArticleHeaderLetterspacingRaw,
+  textStyleArticleHeaderWeightRaw,
+  textStyleArticleRegularFontfamilyRaw,
+  textStyleArticleRegularLetterspacingRaw,
+  textStyleArticleRegularWeightRaw,
+  textStyleCodeHeaderFontfamilyRaw,
+  textStyleCodeHeaderLetterspacingRaw,
+  textStyleCodeHeaderWeightRaw,
+  textStyleCodeRegularFontfamilyRaw,
+  textStyleCodeRegularLetterspacingRaw,
+  textStyleCodeRegularWeightRaw,
+  textStyleDatavizHeaderFontfamilyRaw,
+  textStyleDatavizHeaderLetterspacingRaw,
+  textStyleDatavizHeaderWeightRaw,
+  textStyleDatavizRegularFontfamilyRaw,
+  textStyleDatavizRegularLetterspacingRaw,
+  textStyleDatavizRegularWeightRaw,
+  textStyleDefaultHeaderFontfamilyRaw,
+  textStyleDefaultHeaderLetterspacingRaw,
+  textStyleQuoteHeaderFontfamilyRaw,
+  textStyleQuoteHeaderLetterspacingRaw,
+  textStyleQuoteHeaderWeightRaw,
+  textStyleQuoteRegularFontfamilyRaw,
+  textStyleQuoteRegularLetterspacingRaw,
+  textStyleQuoteRegularWeightRaw,
+} from './variables';
+export const textStyleDefaultHeaderFontfamily = `var(${textStyleDefaultHeaderFontfamilyRaw}, ${textStyleDefaultRegularFontfamilyRaw})`;
+export const textStyleDefaultHeaderLetterspacing = `var(${textStyleDefaultHeaderLetterspacingRaw}, ${textStyleDefaultRegularLetterspacingRaw})`;
+export const textStyleAiRegularFontfamily = `var(${textStyleAiRegularFontfamilyRaw}, ${textStyleDefaultRegularFontfamilyRaw})`;
+export const textStyleAiRegularWeight = `var(${textStyleAiRegularWeightRaw}, ${textStyleDefaultRegularWeightRaw})`;
+export const textStyleAiRegularLetterspacing = `var(${textStyleAiRegularLetterspacingRaw}, ${textStyleDefaultRegularLetterspacingRaw})`;
```

Comment:
Looks like theres's a bug here in the whole file. The second token in the chain should be pointing to the variable string not the raw token string. Credit @marchbox for spotting it.

```diff
- export const tokenA = `var(${tokenARaw}, ${tokenBRaw})`
+ export const tokenA = `var(${tokenARaw}, ${tokenB})`
```
Not sure where else this is a problem.

Reply:
So looking at this, not sure we want to do this actually. We ideally build the template string literal through the algo and raws vs the actual token values. Otherwise, we have a pretty confusing situation and then we also lock tokens into other tokens when we need the chain to be dependent on the specific rule it's applied to. We did have a situation where the vars at the end of the chain didn't have a proper `var()` function call at the end. We are also missing F2 fallbacks for some locations because we haven't defined them in `fluentOverrides.ts` which might be why this seems like it might require using the non-raw version. Those will be filled out as we go though.